frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyNonDraggableItem.tsx

```diff
@@ -0,0 +1,131 @@
+import { useRef } from 'react';
+import { Box, useMediaQuery, useTheme } from '@mui/material';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { StrategySeparator } from 'component/common/StrategySeparator/StrategySeparator';
+import type { IFeatureEnvironment } from 'interfaces/featureToggle';
+import type { IFeatureStrategy } from 'interfaces/strategy';
+import { StrategyItem } from './StrategyItem/StrategyItem';
+import { useRequiredPathParam } from 'hooks/useRequiredPathParam';
+import {
+    useStrategyChangesFromRequest,
+    type UseStrategyChangeFromRequestResult,
+} from './StrategyItem/useStrategyChangesFromRequest';
+import { ChangesScheduledBadge } from 'component/changeRequest/ModifiedInChangeRequestStatusBadge/ChangesScheduledBadge';
+import type { IFeatureChange } from 'component/changeRequest/changeRequest.types';
+import { Badge } from 'component/common/Badge/Badge';
+import {
+    type ScheduledChangeRequestViewModel,
+    useScheduledChangeRequestsWithStrategy,
+} from 'hooks/api/getters/useScheduledChangeRequestsWithStrategy/useScheduledChangeRequestsWithStrategy';
+
+interface IStrategyItemProps {
+    strategy: IFeatureStrategy;
+    environmentName: string;
+    index: number;
+    otherEnvironments?: IFeatureEnvironment['name'][];
+}
+
+/**
+ * @deprecated
+ */
```

Comment:
I decided to just copy the `StrategyDraggableItem` to remove the draggable support. Did not want to pollute the exiting component with more options.

Reply:
What about using existing component and passing no-op functions to drag properties?

Reply:
```
<StrategyDraggableItem
                                    key={strategy.id}
                                    strategy={strategy}
                                    index={index}
                                    environmentName={featureEnvironment.name}
                                    otherEnvironments={otherEnvironments}
                                    isDragging={false}
                                    onDragStartRef={(() => {}) as any}
                                    onDragOver={(() => {}) as any}
                                    onDragEnd={(() => {}) as any}
                                />
```